static/css/components/work.less

```diff
@@ -42,7 +48,7 @@ div.editionAbout {
   }
 }
 
-.meta {
+.meta dt, .meta dd {
```

Comment:
more specific selectors to fix book-page tables font-size

Reply:
Why not just dt and dd. I imagine that should be strong enough